script.js

```diff
@@ -0,0 +1,132 @@
+        computerChoiceSpan.innerText = computersOptions[computerChoice];
+        humanChoiceIcon.innerHTML = '<i class="fa-solid fa-hand fa-5x"></i>';
+        humanChoiceSpan.innerText = "Paper";
+        if (computerChoice === 0) {
+            result.innerText = "You Won!!!";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand-back-fist fa-5x"></i>';
+            humanScore.innerText++;
+        } else if (computerChoice === 1) {
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand fa-5x"></i>';
+            result.innerText = "Draw";
+        } else {
+            result.innerText = "Computer Won!!!";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand-scissors fa-5x"></i>';
+            computerScore.innerText++;
+        }
+    });
+    scissorBtn.addEventListener("click", () => {
+        let computerChoice = random();
+        computerChoiceSpan.innerText = computersOptions[computerChoice];
+        humanChoiceIcon.innerHTML =
+            '<i class="fa-solid fa-hand-scissors fa-5x"></i>';
+        humanChoiceSpan.innerText = "Scissor";
+        if (computerChoice === 0) {
+            result.innerText = "Computer Won!!!";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand-back-fist fa-5x"></i>';
+
+            computerScore.innerText++;
+        } else if (computerChoice === 1) {
+            result.innerText = "You Won!!!";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand fa-5x"></i>';
+            humanScore.innerText++;
+        } else {
+            result.innerText = "Draw";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand-scissors fa-5x"></i>';
+        }
+    });
+    resetBtn.addEventListener("click", () => {
+        reset();
+    });
+};
+
+const random = () => {
+    return Math.floor(Math.random() * 3);
+};
+
+const reset = () => {
+    humanChoiceSpan.innerText = "";
+    computerChoiceSpan.innerText = "";
+    humanScore.innerText = 0;
+    computerScore.innerText = 0;
+    result.innerText = "";
+    humanChoiceIcon.innerHTML = "";
+    computerChoiceIcon.innerHTML = "";
+};
+
+play();
+
+// const object = {
+//     p1: 1,
+//     p2: 2,
+//     p3: 3,
+//     p4: {
+//         p5: 4,
+//     },
+// };
+// console.log(document.getElementsByTagName("p"));
+
+// const prom = new Promise((res, rej) => {
+//     setTimeout(() => {
+//         rej("Working");
+//     }, 10000);
+// });
+
+// console.log(
+//     prom.then(() => console.log("done")).catch(() => console.log("rejected"))
+// );
+
+fetch("https://portal.tycoonstats.com/api/demo")
+    .then((response) => response.text())
+    .then((data) => console.log(data));
+
+async function a() {
+    let response = await fetch("https://portal.tycoonstats.com/api/demo");
+    let data = await response.text();
+    console.log(data);
+}
+
+a();
```

Comment:
1. Make separate commit for different work. 
2. Double check your code before force pushing.
3. Donot push unused / commented out code , make habit of checking the code with "git diff" before updating the commit.